vector-app/build.gradle

```diff
@@ -309,21 +301,11 @@ android {
 //        }
     }
 
-<<<<<<< HEAD
     // The 'store' dimension permits to deal with GooglePlay/Fdroid app
     // The 'target' dimension permits to specify which platform are used
     // The 'voip' flavor dimension permits to include/exclude jitsi at compilation time
     // The 'pinning' flavor dimension permits to enable/disable certificate pinning with fingerprint check
     flavorDimensions "store", "target", "voip", "pinning"
-=======
-    sourceSets {
-        nightly {
-            java.srcDirs += "src/release/java"
-        }
-    }
```

Comment:
please comment these lines and move them upon the flavor dimensions, as it is in Element